apps/ui/src/composables/useActions.ts

```diff
@@ -125,6 +125,8 @@ export function useActions() {
         opts.transactionNetworkId || networkId
       );
     }
+
+    return envelope.trasacton_hash || envelope.hash;
```

Comment:
```suggestion
    return envelope.transaction_hash || envelope.hash;
```

Reply:
Not sure this is always valid, as `envelope` does not always have `hash` and `transaction_hash` properties.

Looking at line 108, we need to fallback to using the hash inside `receipt` instead.

Reply:
On my vote modal PR, I used this fix https://github.com/snapshot-labs/sx-monorepo/pull/627/files#diff-5daf95b187b0b02b358aef78e66671695bdf3e693fd7a9bc9d56068e27bf056e